### PR TITLE
On-ramp: Add NavBar tests to GetStarted and Regions Views

### PR DESCRIPTION
**Pull request by wachunei**
**Development & PR Process**
1. Follow MetaMask [Mobile Coding Standards](https://docs.google.com/document/d/1VJLwTRsUw_5EDq_o8d6sSbXUAYENLurkRitYO45eY5o/edit?usp=sharing)
2. Add `release-xx` label to identify the PR slated for a upcoming release (will be used in release discussion)
3. Add `needs-dev-review` label when work is completed
4. Add `needs-qa` label when dev review is completed
5. Add `QA Passed` label when QA has signed off

**Description**
This PR adds to GetStarted and Regions Views:
- Missing rendering of the Navigation Bar
- Tests of the navigation and analytics events

Bringing coverage to 100%:

_getstartedcoverage
_regionscoverage

**Screenshots/Recordings**

~~_If applicable, add screenshots and/or recordings to visualize the before and after of your change_~~

**Issue**

Progresses #???

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

File: app/components/UI/FiatOnRampAggregator/Views/GetStarted/GetStarted.tsx [hunk 1]
```diff
@@ -120,11 +119,7 @@ const GetStarted: React.FC = () => {
 
       <ScreenLayout.Footer>
         <ScreenLayout.Content>
-          <StyledButton
-            type={'confirm'}
-            onPress={handleOnPress}
-            testID={TEST_ID_GET_STARTED_BUTTON}
```
Comment (sethkfman) [inline app/components/UI/FiatOnRampAggregator/Views/GetStarted/GetStarted.tsx:126]: Why are we removing this test id?

File: app/components/UI/Navbar/index.js [hunk 2]
```diff
@@ -1497,15 +1497,23 @@ export function getFiatOnRampAggNavbar(
       if (!showBack) return <View />;
 
       return Device.isAndroid() ? (
-        <TouchableOpacity onPress={leftAction} style={styles.backButton}>
+        <TouchableOpacity
+          onPress={leftAction}
+          style={styles.backButton}
+          accessibilityRole="button"
```
Comment (sethkfman) [inline app/components/UI/Navbar/index.js:1503]: Do you need to add an accessibility hint? Example:

```
<TouchableOpacity
  accessible={true}
  accessibilityLabel="Go back"
  accessibilityHint="Navigates to the previous screen"
  onPress={onPress}>
  <View style={styles.button}>
    <Text style={styles.buttonText}>Back</Text>
  </View>
</TouchableOpacity>
```

Comment (wachunei) [reply app/components/UI/Navbar/index.js:1503]: We don't need to, but we could, maybe even should! (needs to be translated tho).
I can improve this in an upcoming PR.

Comment (sethkfman) [reply app/components/UI/Navbar/index.js:1503]: Sounds good. Great work on pushing the accessibility effort.

**Review by sethkfman — APPROVED** [review]
LGTM

**Pull request merged**